### ISSUE TITLE
Problem in script

### ISSUE THREAD

**Issue by tutoriel** [Nainterceptor/ENIDownloader#2]
Hi,
There are an issue in the script it doesn't work .
please can you fix it ?

**Comment by Nicolas01** [comment]
Any detail about your issue?  
I used this amazing script few months ago and everything worked fine.

**Comment by Nainterceptor** [comment]
Hello @tutoriel, as explained in #1 (but in french) you've to ask to ENI to enable PDF export option, this option may be disabled on your account. In this case, script cannot work.

**Comment by tutoriel** [comment]
Hi,
@Nicolas01 i don't have any message error . I can download the files but when i open the file i have a message box to insert the password 
@Nainterceptor i have the PDF export option in my account 
Thanks for help

**Comment by tutoriel** [comment]
@Nicolas01 @Nainterceptor  any solution to resolve the problem ?

**Comment by Nainterceptor** [comment]
No, honestly, this project was a one shot project, to help me on one pdf, it's not actively maintained, if you want something, fork & fix by yourself ;)
